### PR TITLE
add moment.js, angular2-cookies, and remove private declaration from variable

### PR DESCRIPTION
**Pull request by amrarick26**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,10 +23,13 @@
     "@angular/router": "^5.0.0",
     "@ng-bootstrap/ng-bootstrap": "^1.0.0-beta.8",
     "@types/lodash": "^4.14.91",
+    "@types/moment": "^2.13.0",
     "angular2-cookie": "^1.2.6",
+    "angular2-cookies": "^0.1.0",
```
Comment (crhistianr) [inline package.json]: why did you add angular2-cookies? Looks like there was already angular2-cookie?

File: src/app/login/login-form/login-form.component.html [hunk 2]
```diff
@@ -22,7 +22,7 @@ <h4 class="card-title text-center">Login</h4>
             <label for="passwordId"
                    class="sr-only">Password</label>
             <input formControlName="password"
-                   type="passwordId"
+                   type="password"
                    class="form-control"
                    placeholder="Password"
                    id="Password" />
```
Comment (crhistianr) [inline src/app/login/login-form/login-form.component.html]: Good catch. can you set the ID to `passwordId` though? This makes sure that screen readers relate the right label to the right input

File: package.json (hunk 1 above)
Comment (amrarick26) [reply package.json]: It was in there from a merge I believe. I never installed it.

File: src/app/login/login-form/login-form.component.html (hunk 2 above)
Comment (amrarick26) [reply src/app/login/login-form/login-form.component.html]: Yeah I can.

**Comment by amrarick26** [comment]
@crhistianr it's updated

**Comment by crhistianr** [comment]
jaysus. we need slack alerts or something never saw this tilll just now
